SDKAndroid/src/main/java/app/cybrid/sdkandroid/util/Pooling.kt

```diff
@@ -0,0 +1,26 @@
+package app.cybrid.sdkandroid.util
+
+import android.os.Handler
+import android.os.Looper
+
+class Pooling(runner: () -> Unit) {
+
+    var updateInterval = 4000L
```

Comment:
This should be made configurable and the default should be a cosntant.

Reply:
It is configurable, in Kotlin dont exists the getter and setter methods, you have to have directly to the property

Then the Kotlin compiler create automatically the setters and getters.

So for example you need to set more time you can do like this:

`var polling = new Polling {}
 polling.updateInterval = 1000L
`